EIPS/eip-5516.md

```diff
@@ -0,0 +1,190 @@
+---
+eip: 5516
+title: Soulbound Multi-owned tokens
+description: An interface for non-transferrable, Multi-owned NFTs binding to Ethereum accounts.
+author: Lucas Martín Grasso Ramos (@LucasGrasso), Matias Arazi (@MatiArazi)
+discussions-to: https://ethereum-magicians.org/t/EIP-5516-soulbound-multi-token-standard/10485
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-19
+requires: 165, 1155
+---
+
+## Abstract
+Proposes a standard interface for non-fungible, double signature, multi-token, Soulbound Tokens. It was inspired by the main characteristics of the [EIP-1155](./eip-1155.md) token and by articles in which benefits and potential use-cases of Soulbound/Accountbound Tokens (SBTs) were presented.
```

Comment:
```suggestion
This EIP proposes a standard interface for non-fungible double signature Soulbound multi-tokens. Previous account-bound token standards face the issue of users losing their account keys or having them rotated, thereby losing their tokens in the process. This EIP provides a solution to this issue and allows for the recycling of SBTs: minting of NFTs with the same characteristics for it being transferred to different addresses is no longer necessary.
```
Recommended: Moved parts of the Motivation to the Abstract
Recommended: Rephrasing